File: C#/백준/Bronze/21638. SMS from MCHS/SMS from MCHS.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            if(bb[0]<0 && bb[1]>=10){
                Console.WriteLine("A storm warning for tomorrow! Be careful and stay home if possible!");
            }
            else if(aa[0]>bb[0]){
                Console.WriteLine("MCHS warns! Low temperature is expected tomorrow.");
            }
            else if(aa[1]<bb[1]){
                Console.WriteLine("MCHS warns! Strong wind is expected tomorrow.");
            }
            else{
                Console.WriteLine("No message");
            }
        }
    }
}

File: Python/백준/Bronze/21638. SMS from MCHS/SMS from MCHS.py
a = list(map(int,input().split()))
b = list(map(int,input().split()))
if(b[0]<0 and b[1]>=10):
    print('A storm warning for tomorrow! Be careful and stay home if possible!')
elif(a[0]>b[0]):
    print('MCHS warns! Low temperature is expected tomorrow.')
elif(a[1]<b[1]):
    print('MCHS warns! Strong wind is expected tomorrow.')
else:
    print('No message')